src/Marten/Linq/SqlGeneration/AnySelectClause.cs

```diff
@@ -25,7 +25,7 @@ public AnySelectClause(string from)
 
         public void WriteSelectClause(CommandBuilder sql)
         {
-            sql.Append("select (count(*) > 0) as result");
+            sql.Append("select 1 = 1 as result");
```

Comment:
Nitpick: `1  = 1` could be changed to `TRUE` to make it clear and explicit.

Reply:
Good Idea, fixed.